src/core/xref.js

```diff
@@ -241,12 +241,24 @@ function addDataCiteToTerms(results, xrefMap, conf) {
 
       // add specs for citation (references section)
       const closestInform = elem.closest(nonNormativeSelector);
-      if (
+      const closestNormative = elem.closest(".normative");
+      let isNormative = false;
+      if (!closestInform) {
+        isNormative = true;
+      } else if (elem === closestNormative) {
```

Comment:
maybe merge this in `if` statement above?
``` js
if (!closestInform || elem === closestNormative) {
  isNormative = true;
} else if (// ...
```

Reply:
Good idea - will add. Could we treat this a nit (i.e., pre-approve this PR)? I’ve got a slight emergency with as this is blocking publications 😓